### DISCUSSION TITLE
支持一下，这个机器人已经很舒服了

### DISCUSSION THREAD

**Discussion by zbhcn** [mod-playerbots/mod-playerbots#189] [General]
支持一下，这个机器人已经很舒服了

**Comment by CoffeeFantasy** [comment]
很棒的机器人！但是我自己不会编译，玩别人的编译机器人不太会使用技能，是不是编译出问题了？

**Comment by oiuv** [comment]
有使用教程吗？我自己编译一切正常，但是在游戏中组队的机器人近战的会上去砍，远程的只会看我打怪，没技能不会用。

**Comment by liyunfan1223** [comment]
谢谢。

@CoffeeFantasy @oiuv 目前要用英文DBC文件才能放技能。

**Comment by oiuv** [comment]
> 谢谢。
> 
> @CoffeeFantasy @oiuv 目前要用英文DBC文件才能放技能。

这个使用英雄DBC能详细解释一下吗？是把服务端配置中的'DBC.Locale'改成0，还是要用英文的客户端来生成dbc？

**Comment by oiuv** [comment]
@liyunfan1223 服务端是不是要更新一下呀，同步最新的呀。

This branch is [227 commits ahead of](https://github.com/liyunfan1223/azerothcore-wotlk/compare/azerothcore%3Aazerothcore-wotlk%3Amaster...Playerbot), [8 commits behind](https://github.com/liyunfan1223/azerothcore-wotlk/compare/Playerbot...azerothcore%3Aazerothcore-wotlk%3Amaster) azerothcore/azerothcore-wotlk:master.

**Comment by oiuv** [comment]
我下载了一个enUS的客户端，重新生成dbc，开启游戏测试了下，可以正常的使用spell了

**Comment by alexyang-sea** [comment]
DBC.Local用默认255就可以，客户端中英文都没关系，我是用的中文的

**Comment by CoffeeFantasy** [comment]
> 有使用教程吗？我自己编译一切正常，但是在游戏中组队的机器人近战的会上去砍，远程的只会看我打怪，没技能不会用。

你可以尝试下更改bot的编辑策略，我之前的猎人bot也只是平射更改编辑策略后恢复正常

**Comment by CoffeeFantasy** [comment]
目前自己学会了下编译，很看好这个版本的机器人。对比天蓝版本拓展性更强，问题就是上手难了点，在liyunfan大佬的优化下肯定会越来越棒

**Comment by alexyang-sea** [comment]
> 目前自己学会了下编译，很看好这个版本的机器人。对比天蓝版本拓展性更强，问题就是上手难了点，在liyunfan大佬的优化下肯定会越来越棒

确实，在liyunfan大佬的开发下，机器人是越来越好啦，期待未来优化越来越好，AI越来越智能，打团本打战场像跟真人玩家那样就牛啦，未来可期

**Comment by shenk-co** [comment]
> 我下载了一个enUS的客户端，重新生成dbc，开启游戏测试了下，可以正常的使用spell了

只要改一下服务器端的data中的spell.dbc就好了，网上有添加了中文的英文spell.dbc 的

**Comment by ljzam** [comment]
最近发现随机机器人的技能不全，比如暗牧，用unbot查看了下技能，很多暗牧天赋的技能都没有，导致输出异常低。请问是什么问题？
